include/translatable-object.php

```diff
@@ -372,14 +396,13 @@ public function get_objects_with_no_lang( $limit, array $args = array() ) {
 		}
 
 		$key          = md5( $sql );
-		$cache_type   = "{$this->type}s";
-		$last_changed = wp_cache_get_last_changed( $cache_type );
-		$cache_key    = "{$cache_type}_no_lang:{$key}:{$last_changed}";
-		$object_ids   = wp_cache_get( $cache_key, $cache_type );
+		$last_changed = wp_cache_get_last_changed( $this->cache_type );
+		$cache_key    = $this->get_cache_key( $key, $last_changed );
```

Comment:
```suggestion
		$cache_key    =  "{$this->cache_type}_no_lang:{$key}:{$last_changed}";
```
I suggest to revert the introduction of the function.
1. The code is short and cache key is used only once so introducing a new function is not useful.
2. The function name is too generic (the cache key is for a specific SQL query).